### DISCUSSION TITLE
评论hexo d 后无法用

### DISCUSSION THREAD

**Discussion by taotangSeek** [YunYouJun/hexo-theme-starry#241] [Q&A 疑难杂症]
我也非常喜欢您的主题哈~~
我问一下，再本地部署了评论后，再部署到github后，进行评论后，无法进行，跳出forbidden的提示：
![image](https://user-images.githubusercontent.com/76051101/149648389-cdce5d11-d5c6-4594-b54d-924bd1e7101c.png)
请问这是怎么回事呢：

**Comment by YunYouJun** [comment]
请说明您所使用的评论系统。

其次请查看对应评论系统官方文档（比如可能需要申请 appKey），根据对应说明进行部署。

最后确认是否已将自己的域名添加至安全域名。
